app/views/admin/merchants/show.html.erb

```diff
@@ -0,0 +1,15 @@
+
+<h1><%= @merchant.name %></h1>
+
+<h3><%= @merchant.address %> <%= @merchant.city %> <%= @merchant.state %> <%= @merchant.zip %></h3>
+
+<p><%=link_to "My Items", "/merchant/items" %></p>
+
+<ul>
+  <% @merchant.orders.each do |order| %>
+    <li>Order: <%= link_to order.id, "/orders/#{order.id}"%></li>
+    <li>Quantity: <%= order.total_quantity %></li>
+    <li>Subtotal: <%= order.grandtotal %></li>
+    <li>Order Date: <%= order.created_at.to_s[0..9] %></li><br>
+  <% end %>
+</ul>
```

Comment:
Since this view should be exactly like the `merchant/dashboard/index.html.erb` view do we need to create this one or can we just render that same view from the admin controller? Not sure if this violated any authorization rules.

If the above is not possible then maybe we could link both pages to the same partial. Thoughts?

Reply:
I think a partial would work in this case. I'm not sure if there would be any way to specify that both routes go to the same view without breaking authorization/shuffling files out of our admin/merchant view directories.

Reply:
I guess because we have to check for an authorization, a partial would be the best way to move forward without exactly replicating each show page.  One of my questions would be is where in the directory do we want to store the `_main_html_doc` so that it each show page can still double check in their particular render has a specific user and follow the routes needed to make it work.